.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
quite a file. Would it be possible to only have the variables inside which are not default? This makes it easier to spot where we don't follow the default behaviour.